tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -740,8 +740,30 @@ test_i2c_tcs3400: tcs3400@72 {
 	int-gpios = <&test_gpio 0 0>;
 };
 
-test_tcn75a: tcn75a@73 {
+test_tcn75a: tcn75a@74 {
 	compatible = "microchip,tcn75a";
-	reg = <0x73>;
+	reg = <0x74>;
```

Comment:
I believe this maybe got messed up in the rebase? I think it should be 73 not 74 from this [commit](https://github.com/zephyrproject-rtos/zephyr/commit/d3828418c09b2f35463da841a893798bc4ab9ce7): https://github.com/zephyrproject-rtos/zephyr/blob/fed377f622c9c280016c41b22ea6dc6b8ce0aed4/tests/drivers/build_all/sensor/i2c.dtsi#L743-L745